Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2464,7 +2463,6 @@ extension BrowserViewController: TabDelegate {
     KVOs.forEach { webView.removeObserver(self, forKeyPath: $0.rawValue) }
     toolbarVisibilityViewModel.endScrollViewObservation(webView.scrollView)
     webView.uiDelegate = nil
-    webView.scrollView.delegate = nil
```

Comment:
This is not part of the fix, just redundant as this is done in `endScrollViewObservation` above